Jenkinsfile

```diff
@@ -0,0 +1,110 @@
+pipeline {
+  agent {
+    dockerfile true
+  }
+  environment {
+    def NPM_CONFIG_CACHE = '/tmp/npm'
+    def info = repoInfo()
+  }
+  stages {
+    stage('prepare') {
+      when {
+        branch 'master'
+      }
+      steps {
+        script {
+          env.CI_SKIP = "false"
+          result = sh (script: "git log -1 | grep '.*\\[ci auto incrementing version\\].*'", returnStatus: true)
+          if (result == 0) {
+              env.CI_SKIP = "true"
+              error "'[ci auto incrementing version]' found in git commit message. Aborting."
+          }
+        }
+      }
+    }
+    stage('bump version') {
+      when {
+        branch 'master'
+      }
+      steps {
+        script {
+          sh 'npm --no-git-tag-version version minor'
+        }
+      }
+    }
+    stage('publish') {
+      when {
+        branch 'master'
+      }
+      steps {
+        script {
+          withVaultCredentials([
+            [path: '/secret/content-engineering/global/artifactory/content-engineering@gannett.com', keys: ['username': 'ARTIFACTORY_USERNAME', 'password': 'ARTIFACTORY_PASSWORD']],
+            [path: "/secret/content-engineering/global/tyk/paas-api", keys: ['TYK_API_KEY': 'TYK_API_KEY']],
```

Comment:
we are no longer using TYK_API, change it to apigee. The path is ```/secret/content-engineering/global/apigee/paas-api` 
keys: ```['API_KEY': 'API_KEY']],```

Reply:
Good call, will update.